### ISSUE TITLE
Compose V2 - Node hanging while waiting for xvfb

### ISSUE THREAD

**Issue by paul-nelson-baker** [SeleniumHQ/docker-selenium#407]
## Meta -
Image(s):  Not applicable.

Docker Version:  
```
Client:
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 08:47:51 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 08:47:51 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

OS: `OSX Sierra`

## Expected Behavior -
Given this `docker-compose.yml` file, I should be able to create a local grid on my machine and node-chrome-debug should be able to instantiate itself and register with hub.

```
version: '2.0'
services:
  grid-hub:
    image: 'selenium/hub'
    ports:
      - '4444:4444'
  node-chrome-debug:
    image: 'selenium/node-chrome-debug'
    depends_on:
      - 'grid-hub'
    environment:
      - HUB_PORT_4444_TCP_ADDR=grid-hub
      - HUB_PORT_4444_TCP_PORT=4444
```

## Actual Behavior -
The node hangs and never becomes visible to me from the grid console (http://localhost:4444/grid/console)

The output of node-chrome-debug is only:
```
Waiting xvfb...
-bash: 169.254/16: No such file or directory
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
```

I noticed a [similar issue](https://github.com/SeleniumHQ/docker-selenium/issues/401), but to ensure that there are no conflicting images or containers I have deleted all local data and seen no difference. (This was done by running `docker rm $(docker ps -a -q)` and `docker rmi $(docker images -q)`)

UPDATE: This might be more relevant to [this issue](https://github.com/SeleniumHQ/docker-selenium/pull/249).

**Comment by paul-nelson-baker** [comment]
I have a temporary fix. Based on the comment by `ddavison` from the [linked pull request](https://github.com/SeleniumHQ/docker-selenium/pull/249)

```
version: '3.0'
services:
  grid-hub:
    container_name: 'grid-hub'
    image: 'selenium/hub'
    ports:
      - '4444:4444'
  node-chrome-debug:
    image: 'selenium/node-chrome-debug'
    depends_on:
      - 'grid-hub'
    environment:
      - HUB_PORT_4444_TCP_ADDR=grid-hub
      - HUB_PORT_4444_TCP_PORT=4444
      - no_proxy=""
```

Is it more appropriate to close this issue, or to leave it open until the pull request is merged?

**Comment by diemol** [comment]
@paul-nelson-baker I just tried your docker-compose file with the following info:

* docker version info:
```
Client:
 Version:      17.03.1-ce
 API version:  1.27
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Tue Mar 28 00:40:02 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      17.03.1-ce
 API version:  1.27 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Fri Mar 24 00:00:50 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

* docker-selenium images (these come from the latest release 3.3.1-Cesium):

```
REPOSITORY                   TAG                 IMAGE ID            CREATED             SIZE
selenium/node-chrome-debug   latest              08799a1d9bf4        4 days ago          858 MB
selenium/hub                 latest              8b117008694b        4 days ago          312 MB
``` 

* OS Version:

```
macOS Sierra 10.12.4 (16E195)
```

and it worked right away for me. Could you please pull the latest images and check again? It should work with your current docker version.

**Comment by diemol** [comment]
This was fixed by #459, it can be closed.

**Comment by diemol** [comment]
@ddavison, can you please close this one?
